getting_started/first_3d_game/02.player_input.rst

```diff
@@ -149,8 +149,7 @@ your interface should look like this.
 
 |image15|
 
-The final action to set up is the ``jump`` action. Bind the Space key and the gamepad's
-A button.
+The final action to set up is the ``jump`` action. Bind the Space key and joypad Button 0.
```

Comment:
```suggestion
The final action to set up is the ``jump`` action. Bind the Space key and
Joypad Button 0 (Bottom Action, Sony Cross, Xbox A, Nintendo B).
```

Reply:
Indeed, `A` might be the internal name it is not universal and using it is misleading, providing a list of the proper names would be the best solution

Reply:
Sorry, I could've been more descriptive. I made an update to list the buttons that button 0 is mapped to.